service/kvmconfig/drainer_framework.go

```diff
@@ -43,7 +38,7 @@ func NewFramework(config FrameworkConfig) (*framework.Framework, error) {
 			Watcher: config.K8sClient.CoreV1().Pods(""),
 
 			ListOptions: apismetav1.ListOptions{
-				LabelSelector: fmt.Sprintf("%s=%s", PodWatcherLabel, config.ProjectName),
+				LabelSelector: fmt.Sprintf("%s=%s", key.PodWatcherLabel, config.ProjectName),
```

Comment:
So the cool thing is everything is versioned together and the key package can be reused across the resources without diverging further.